edit/edit.js

```diff
@@ -1744,12 +1747,17 @@ function showKeyMapHelp() {
     const col = input.parentNode.cellIndex;
     inputs[1 - col].value = '';
     table.tBodies[0].childNodes.forEach(row => {
-      let cell = row.children[col];
-      cell.innerHTML = cell.textContent.replace(query, '<mark>$&</mark>');
+      const cell = row.children[col];
+      const html = cell.textContent.replace(query, '<mark>$&</mark>');
+      cell.textContent = '';
+      const div = tHTML(html, 'div');
+      while (div.childNodes.length > 0) {
+        cell.appendChild(div.childNodes[0]);
+      }
       row.style.display = query.test(cell.textContent) ? '' : 'none';
       // clear highlight from the other column
```

Comment:
I am not following. `otherCell.textContent = otherCell.textContent` does nothing.